README.md

```diff
@@ -7,10 +7,10 @@
   </a>
 </p>
 <p align="center">
-  <a href="https://github.com/lucide-icons/lucide/blob/main/LICENSE"><img src="https://img.shields.io/npm/l/lucide" alt="license"></a>
-  <a href="https://www.figma.com/community/plugin/939567362549682242/Lucide-Icons"><img src="https://img.shields.io/badge/Figma-F24E1E?logo=figma&logoColor=white" alt="figma installs"></a>
-  <a href="https://github.com/lucide-icons/lucide/actions/workflows/release.yml"><img src="https://github.com/lucide-icons/lucide/actions/workflows/release.yml/badge.svg" alt="build status"></a>
-  <a href="https://discord.gg/EH6nSts"><img src="https://img.shields.io/discord/723074157486800936?label=chat&logo=discord&logoColor=%23ffffff&colorB=%237289DA" alt="discord chat"></a>
+  <a href="https://github.com/lucide-icons/lucide/blob/main/LICENSE"><img src="https://img.shields.io/npm/l/lucide?style=for-the-badge" alt="license"></a>
```

Comment:
```suggestion
  <a href="https://github.com/lucide-icons/lucide/blob/main/LICENSE"><img src="https://img.shields.io/npm/l/lucide?style=for-the-badge&color=008c14" alt="license"></a>
```
I like @JoshuaKGoldberg s suggestion of changing the background colors to be a bit darker.
https://github.com/badges/shields/issues/5497

What do you think?

Reply:
Yeah, that feels more right. I'll do it now

Reply:
I feel like `#008c14` gets too dark, another option is to use `brightgreen` which looks like this:
![image](https://github.com/user-attachments/assets/83bdf376-c3b6-422c-a9fe-6bc158ba2978)

Reply:
Minimum contrast to get me to be happy is 4.5.

Reply:
Will fix it when i get back my changes which I of course lost

Reply:
`#1A871B`

Reply:
or should we just use `brightgreen`/`#45CC11` as the color and make the foreground black

Reply:
Possible colors:
![image](https://github.com/user-attachments/assets/56b1c859-5213-410a-adef-d2a0f7ac5732)

![image](https://github.com/user-attachments/assets/2897eae8-f1b8-4b87-81e2-5ca11cadea19)

![image](https://github.com/user-attachments/assets/1b3aa044-7ab0-42eb-9c6d-0168bc18eeaa)

![image](https://github.com/user-attachments/assets/720be9be-5fe3-4919-95cc-cef9d720c4c7)

Reply:
We could also make all the badges the lucide brand color, still pretty low tho.

Reply:
Yeah, red and white, what badge should have which

Reply:
Or should we even care about the contrast, there's a bunch (and I mean a bunch) using the `brightgreen` for example, which isn't contrast sufficient enough

Reply:
If it's not necessary to be readable, it's not necessary and should be removed entirely, I'm pretty strongly opinionated on this one :)

Reply:
Found some colors with high enough (very close at least) contrast ratio. I'm not updating the other ones until we've found the final colors

![image](https://github.com/user-attachments/assets/c725ba2c-df2d-4ea3-95a2-3bbc9115dfb0)